packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html

```diff
@@ -12,7 +12,7 @@
                 id="v"
                 name="v"
                 data-model="ctrl.model.target.v.DKIM1"
-                data-on-change="ctrl.setTargetValue('DKIM')"
+                data-on-change="ctrl.model.target.v.DKIM1 = modelValue; ctrl.setTargetValue('DKIM')"
```

Comment:
can you add this in controller and call that method in all other places plz, also add 'DKIM' in contants

Reply:
There are a lot of constants and the whole code needs to be rewritten in future so we should do it when we refactor :)

Reply:
Ok @varun257 but please create a JIRA for this and add it to `TO_GROOM` sprint, this way we don't forget about this 🙂